coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -115,15 +115,16 @@ public RelToAvroSchemaConverter(HiveMetastoreClient hiveMetastoreClient) {
    *
    * @param relNode {@link RelNode} object
    * @param strictMode configure whether to use strict mode
+   * @param forceLowercase  configure whether to force lowercase
    * @return avro schema for calcite IR RelNode
    * @throws RuntimeException if cannot find table in Hive metastore
    * @throws RuntimeException if cannot determine avro schema for tableScan
    */
-  public Schema convert(@Nonnull RelNode relNode, boolean strictMode) {
+  public Schema convert(@Nonnull RelNode relNode, boolean strictMode, boolean forceLowercase) {
```

Comment:
I'm also not sure if there are direct consumers of this method outside Coral. However, since it is a public API, it makes sense to keep previous signature for backward compatibility. Kept `Schema convert(@Nonnull RelNode relNode, boolean strictMode)` and made it call the new API internally.